src/pages/CalendarView.jsx

```diff
@@ -4,11 +4,11 @@ import 'react-calendar/dist/Calendar.css';
 import ActivityOption from '../components/ActivityOption';
 import DateInformation from '../components/DateInformation';
 import ContactOption from '../components/ContactOption';
+import FrequencyOption from '../components/FrequencyOption';
 
 const BASE_URL = '/api/v1/events';
-const FREQUENCY = ['Once', 'Daily', 'Weekly', 'Biweekly', 'Monthly'];
 
-function CalendarView() {
+export default function CalendarView() {
```

Comment:
I prefer putting `export default CalendarView;` as a separate statement. This is especially needed for props validation.

Reply:
That makes more sense!
I made a new commit for it